docs/ops/state/state_backends.md

```diff
@@ -110,10 +110,10 @@ Limitations of the RocksDBStateBackend:
 
 The RocksDBStateBackend is encouraged for:
 
-  - Jobs with very large state, long windows, large key/value states.
+  - Jobs with a very large state, long windows, large key/value states.
```

Comment:
I'm not entirely sure that this change is necessary, but may be wrong as I'm not a native English speaker :)

Reply:
This change is ungrammatical, but the rest are good.